src/js_parser.zig

```diff
@@ -8969,6 +8990,8 @@ fn NewParser_(
                         .import_record_index = stmt.import_record_index,
                     }) catch unreachable;
                 }
+
+                // TODO: not sure how to handle macro remappings for namespace imports
```

Comment:
I’m not too sure about how the package.json macro remapping feature works. I tried to do what I can, but to allow namepspace imports to be remapped would probably be pretty hard as far as I can tell.